### ISSUE TITLE
Empty plots produced

### ISSUE THREAD

**Issue by fedarko** [fedarko/rankratioviz#41]
I think this is due to a mismatch of input data somehow -- e.g. samples in the biom table don't relate to the metadata file, or features in the biom table don't relate to the taxonomy file, or ranks in the ordination file don't relate to features.

I'm experiencing this with the Byrd skin data when thrown into Q2, and in other cases. I'll try to figure out what's behind this.

**Comment by fedarko** [comment]
It looks like the ordination file for the nonworking case (which you can look at by just unzipping the relevant qza and looking in data/ordination.txt) just has three features? which seem to correspond to the headers of the beta.csv file (intercept, c(timepoint, treatment('F'))[T.B], ...).

Whereas the working ordination file has 609 features and 184 samples.

Actually... it looks like features and samples were swapped around in the nonworking ordination file. The "samples" are all just taxa names in the same sorta style as in beta.csv.

In the process of generating the nonworking ordination file, I ran `qiime songbird multinomial` then `qiime songbird regression-biplot`. I remember that QIIME 2 complained about the ordination file, actually -- 

```
There was an issue with loading the file /Users/mfedarko/skin-metagenome/data/byrd_metadata.txt as metadata:

  Found unrecognized ID column name 'Sample_ID' while searching for header. The first column name in the header defines the ID column, and must be one of these values:

  Case-insensitive: 'feature id', 'feature-id', 'featureid', 'id', 'sample id', 'sample-id', 'sampleid'

  Case-sensitive: '#OTU ID', '#OTUID', '#Sample ID', '#SampleID', 'sample_name'

  There may be more errors present in the metadata file. To get a full report, sample/feature metadata files can be validated with Keemei: https://keemei.qiime2.org

  Find details on QIIME 2 metadata requirements here: https://docs.qiime2.org/2018.11/tutorials/metadata/

```

And q2-emperor didn't like it even when I adjusted the metadata file to use an accepted sample-id column name:

```
Plugin error from emperor:

  None of the sample identifiers match between the metadata and the coordinates. Verify that you are using metadata and coordinates corresponding to the same dataset.
```

So this is probably an error with the metadata somehow. I'll try to get it fixed ASAP.

**Comment by fedarko** [comment]
After a few days of working on this + conversations with Jamie, I'm like 99% sure this will be resolved when #51 is (it's a problem with handling inputs from songbird).

However, I'm leaving it open for now just in case it isn't for some reason.

**Comment by fedarko** [comment]
Yeah, the Byrd data works in rrv now. Closing.
